mmv1/third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -2685,6 +2711,16 @@ func testAccCheckComputeInstanceHasNetworkIP(instance *compute.Instance, network
 	}
 }
 
+func testAccCheckComputeInstanceHasNetworkPerformanceConfig(instance *computeBeta.Instance, bandwidthTier string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if instance.NetworkPerformanceConfig.TotalEgressBandwidthTier != bandwidthTier {
```

Comment:
Can we place a nil check for `instance.NetworkPerformanceConfig` ?
If `instance.NetworkPerformanceConfig` is nil, we should error out before checking `instance.NetworkPerformanceConfig.TotalEgressBandwidthTier`. Just in case.

Reply:
Yes, sure thing.